source/process/guest-article.rst

```diff
@@ -0,0 +1,21 @@
+Guest Articles for Mattermost Apps and Services
```

Comment:
"for Mattermost Apps and Services" was suggested by Ian and I decided to leave it.